### PR TITLE
Add action_class block documentation

### PR DESCRIPTION
**Pull request by MarkGibbons**
Use ::File instead of File in the example code.

I hope the action_class block doc makes sense.  It's based more on how I've been using action class than the code..

### CODE REVIEW

File: chef_master/source/custom_resources.rst [hunk 1]
```diff
@@ -489,7 +489,7 @@ When finished adding the templates and building the custom resource, the cookboo
 
 Recipe
 -----------------------------------------------------
-The custom resource name is inferred from the name of the cookbook (``website``), the name of the recipe (``httpd``), and is separated by an underscore(``_``): ``website_httpd``.
```
Comment (tas50) [inline chef_master/source/custom_resources.rst:492]: Well that was painfully wrong

**Comment by tas50** [comment]
You beat me to it. I kept going to write this and I got hung up on other issues in the docs repo. This is a welcome addition to the docs though. Thanks for putting it together.

**Comment by MarkGibbons** [comment]
I would have written it sooner but it's pretty hard to figure out how it works.  Thanks for your help.

**Comment by davidwrede** [comment]
@MarkGibbons This is great! 

gif-keyboard-13815866787879495116

**Pull request merged**